packages/imperative/src/imperative/src/config/cmd/report-env/EnvItems.ts

```diff
@@ -68,8 +68,8 @@ function formatLogLevelMsg(logTypeName: string) {
 export const probTests: IProbTest[] = [
     {
         itemId: ItemId.NODEJS_VER,
-        probExpr: "semver.satisfies('{val}', '<18.x || 19.x || >=21.x')",
-        probMsg: "Only Node.js versions 18 and 20 are supported."
+        probExpr: "semver.satisfies('{val}', '<18.x || 19.x || 21.x || >=23.x')",
```

Comment:
Thanks for updating this semver expression, I was also noticing this test fail locally 😍